src/app-view.js

```diff
@@ -52,48 +42,38 @@ class AppView extends React.Component {
 
     render() {
         return (
-            <AppViewComponent>
-                <AppViewGlobalStyles />
-                <Sidebar
-                    isVisible={this.state.isSidebarVisible}
-                    onClickToggle={this.handleSidebarToggleClick}
-                >
+            <AppViewComponent
+                searchField={
                     <SearchField
                         value={this.props.searchQuery}
                         onChange={this.props.onSearchFieldChange}
                     />
+                }
+                navigation={
                     <Navigation
                         expandAll={!!this.props.searchQuery}
                         currentHash={this.props.currentHash}
                         sections={this.state.sections}
                     />
-                </Sidebar>
-                <Content isExpanded={!this.state.isSidebarVisible}>
-                    {this.state.component ? (
-                        <Section content={this.state.component} />
-                    ) : (
-                        <Component
-                            name={'Welcome!'}
-                            description={
-                                'Style guide is a tool to illustrate, sandbox and test your components.'
-                            }
-                        />
-                    )}
-                </Content>
-            </AppViewComponent>
+                }
+                content={
+                    <Content>
+                        {this.state.component ? (
+                            <Section content={this.state.component} />
+                        ) : (
+                            <Component
+                                name={'Welcome!'}
```

Comment:
name={'Welcome!'}
name="Welcome!"

Reply:
description="Style guide is a tool to illustrate, sandbox and test your components."